api/views.py

```
@@ -0,0 +1,1021 @@
+        appeal_id = serializer.validated_data["appeal_id"]
+        appeal = get_object_or_404(
+            Appeal.filter_to_allowed_appeals(current_user), pk=appeal_id
+        )
+        if serializer["fax_number"] is not None:
+            appeal.fax_number = serializer["fax_number"]
+            appeal.save()
+        patient_user = None
+        try:
+            patient_user = PatientUser.objects.get(user=current_user)
+        except:
+            pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)